deploy/2.mocks.ts

```diff
@@ -90,6 +90,17 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     await trx.wait();
   }
 
+  // set initial borrow index
+  let trx = await mockCToken.setBorrowIndex(BigNumber.from(10).pow(18));
+  await trx.wait();
+
+  const blocksPerYear = BigNumber.from(31536000).div(13);
+  const wad = BigNumber.from(10).pow(18);
```

Comment:
@ioanaroceanu2000 can you pls confirm that these numbers are indeed supposed to be in wad and are not a function of the number of decimals of the underlying token?

Reply:
https://github.com/compound-finance/compound-protocol/blob/a3214f67b73310d547e00fc578e8355911c9d376/contracts/CToken.sol#L205